lighthouse-core/gather/connections/connection.js

```diff
@@ -106,11 +106,15 @@ class Connection {
           {method: callback.method, params: object.result}, 'verbose');
         return object.result;
       }));
+    } else if (object.id) {
+      const error = object.error && object.error.message;
+      log.formatProtocol(`disowned method <= browser ${error ? 'ERR' : 'OK'}`,
```

Comment:
`disowned` may not be a good word here (sounds like LH did send the command but doesn't care about it now :), but not sure of a good replacement.